AUTHORS.md

```diff
@@ -0,0 +1,3 @@
+Sachin Argade, doubleSlash Net-Business GmbH, https://github.com/sachinargade123
```

Comment:
Please format file correctly and follow example content mentioned in [TRG 2-03](https://eclipse-tractusx.github.io/docs/release/trg-2/trg-2-3#authorsmd-optional).

Reply:
Corrected the format.

Reply:
The format of this file is still broken and contains no valid markdown in some way. Compare your [Author.md](https://github.com/catenax-ng/tx-dft-backend/blob/main/AUTHORS.md) with [this one](https://github.com/eclipse-tractusx/charts/blob/main/AUTHORS.md) then it should be obvious what to do.

Reply:
please reverify changes

Reply:
Finally.